staging/src/k8s.io/apiserver/pkg/storage/cacher/store.go

```diff
@@ -39,8 +41,16 @@ type storeIndexer interface {
 	ByIndex(indexName, indexedValue string) ([]interface{}, error)
 }
 
+type orderedLister interface {
+	ListPrefix(prefix, continueKey string, limit int) (items []interface{}, hasMore bool)
+}
+
 func newStoreIndexer(indexers *cache.Indexers) storeIndexer {
-	return cache.NewIndexer(storeElementKey, storeElementIndexers(indexers))
+	if utilfeature.DefaultFeatureGate.Enabled(features.BtreeWatchCache) {
+		return newThreadedBtreeStoreIndexer(storeElementIndexers(indexers), 32)
```

Comment:
Let's highlight `32` as a constant please?

Reply:
DOne

Reply:
thank you! especially for adding more notes on how you got that number